test/unit/helpers/views/TransportMessages.scala

```diff
@@ -19,6 +19,7 @@ package helpers.views
 trait TransportMessages {
 
   val title = "transport.title"
+  val heading = "transport.heading"
```

Comment:
We are tending to get rid of / not use these message helper classes - just reference the message keys directly in the test (which you appear to have done)